plugins/base.py

```diff
@@ -30,11 +30,13 @@ def on_load_end(self):
         function_origin = type(self).on_load_end.__qualname__.split(".")[0]
         print_checker(function_origin, f"{self.printprefix} on_load_end")
 
-    def on_fit_begin(self, store: Store, last_output: Any) -> Tuple[Store, Any]:
+    def on_fit_begin(
+        self, store: Store, last_output: Any
```

Comment:
can we create a type for this? OnFitCallbacks = List[Callable] or whatever, it's impossible to decipher what List[Callable] is otherwise!

Reply:
it can be anything at all if we don't specify it

Reply:
or List[OnFitCallback]